vocabulary/debugging-terminology.md

```diff
@@ -0,0 +1,41 @@
+# Some Debugging Terminology
```

Comment:
Hey, @Darin-Hamouda. This is al correct but has little to do with debugging  web applications(except the debugger part).  All the other terms have to do with operating systems and are not that interesting for this course. You could try to find some other terms that are more focused on web apps and consider removing the ones that are less useful?